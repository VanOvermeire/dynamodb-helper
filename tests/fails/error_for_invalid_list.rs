use dynamodb_helper::DynamoDb;

#[derive(DynamoDb)]
pub struct Example {
    #[partition]
    first: String,
    second: u32,
    invalid: Vec<bool>,
}

fn main() {}
