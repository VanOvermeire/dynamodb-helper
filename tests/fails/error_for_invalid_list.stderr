error: Only lists with strings or numbers are supported
 --> tests/fails/error_for_invalid_list.rs:8:5
  |
8 |     invalid: Vec<bool>,
  |     ^^^^^^^
